src/main/java/com/hms_networks/sc/cumulocity/api/CConnectorApiCertificateMgr.java

```diff
@@ -0,0 +1,101 @@
+package com.hms_networks.sc.cumulocity.api;
+
+import com.ewon.ewonitf.EWException;
+import com.ewon.ewonitf.ScheduledActionManager;
+import com.hms_networks.americas.sc.extensions.logging.Logger;
+import com.hms_networks.americas.sc.extensions.system.http.SCHttpUtility;
+import java.io.File;
+
+/**
+ * Utility class for managing the download and storage of the root CA certificate for Cumulocity
+ * (GoDaddy Class 2 Certification Authority Root Certificate).
+ *
+ * @author HMS Networks, MU Americas Solution Center
+ * @since 1.0.0
+ */
+public class CConnectorApiCertificateMgr {
+
+  /** The host which serves the Cumulocity Root CA. */
+  public static final String CUMULOCITY_ROOT_CA_DOWNLOAD_HOST = "https://certs.godaddy.com";
+
+  /** The path to download the Cumulocity Root CA. */
+  public static final String CUMULOCITY_ROOT_CA_DOWNLOAD_PATH = "/repository/gdroot-g2.crt";
```

Comment:
This certificate is valid until 2037 and is the root so changes to intermediate certificates in the meantime should not affect this. Additionally, the GoDaddy certificate repository is also where revocation lists and certificate problem reporting (compromise, misuse, etc) are located, so it shouldn't ever be moved from this URL.

In my opinion, adding this to the configuration file is only going to add a potential confusion point for something that should not change. Additionally, it is a field that, when misconfigured, set improperly, or accidentally changed by a user, could break their entire connection to Cumulocity and lead to increased support requests.

Reply:
ok, you are right... the probability that customers get the configuration option is wrong is far more likely than customers complain because we need to rebuild because of rootCA change.